src/azure-cli/azure/cli/command_modules/storage/_params.py

```diff
@@ -154,6 +154,9 @@ def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statem
         c.argument('domain_guid', domain_guid_type)
         c.argument('domain_sid', domain_sid_type)
         c.argument('azure_storage_sid', azure_storage_sid_type)
+        c.argument('enable_hierarchical_namespace', options_list=['--enable-hierarchical-namespace', '--hns'],
+                   action='store_true', help=" Allow the blob service to exhibit filesystem semantics",
```

Comment:
file system

Reply:
What's the type of it?

Reply:
Can we disable the hierarchical_namespace?

Reply:
> Can we disable the hierarchical_namespace?

Default is disabled.

Reply:
> What's the type of it?

boolean

Reply:
> 
> 
> > What's the type of it?
> 
> boolean

Do we need a `arg_type=get_three_state_flag()`?